CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 5.0.0-beta.2 (2021-09-14)
```

Comment:
Should this be `5.0.0-beta.1` since that's what's in `package.json`?

Reply:
ugh. typo. good catch.

Reply:
done. That was a test. You passed.